test/es-module/test-esm-exports.mjs

```diff
@@ -44,6 +44,10 @@ import fromInside from '../fixtures/node_modules/pkgexports/lib/hole.js';
     validSpecifiers.set('pkgexports/subpath/dir1/', { default: 'main' });
     validSpecifiers.set('pkgexports/subpath/dir2', { default: 'index' });
     validSpecifiers.set('pkgexports/subpath/dir2/', { default: 'index' });
+  } else {
+    // no exports field
+    validSpecifiers.set('no_exports', { default: 'main' });
```

Comment:
```suggestion
    validSpecifiers.set('no_exports', { default: 'index' });
```

Reply:
It's called "main" in the test itself, so this is required for it to pass.

Reply:
Ahh, GitHub was hiding the other suggestion, adding both.